docs/client/install-kn.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing the kn CLI"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/install/install-kn
+  - /docs/client/connecting-kn-to-your-cluster
+---
+
+This guide provides details about how you can install the Knative `kn` client (CLI) using various methods.
+
+## Install kn using an executable binary
```

Comment:
kn is always an executable binary. I think the point here is "by downloading from GitHub", so I would add this as title.

Reply:
Not really, as a user I wouldn't care whether this was downloaded from github or somewhere else, the important part to me is that it's a binary so I know I can just download it and move it to the path and that's it. We don't actually even mention github in this section so to me it doesn't make sense as a title.